src/Telegram.Bot/Types/InputFiles/InputFileUrl.cs

```diff
@@ -4,21 +4,28 @@
 namespace Telegram.Bot.Types;
 
 /// <summary>
-/// A file identifier
+/// This object represents an HTTP URL for the file to be sent
 /// </summary>
 public class InputFileUrl : IInputFile
 {
     /// <inheritdoc/>
     public FileType FileType => FileType.Url;
 
     /// <summary>
-    ///
+    /// HTTP URL for the file to be sent
     /// </summary>
     public string Url { get; }
```

Comment:
I think we should just make it a Uri instance and accept it everywhere we accepted strings as URIs

Reply:
we don't have any `Uri` typed properties in any other classes though